.github/workflows/tests.yaml

```diff
@@ -39,18 +39,18 @@ jobs:
       - name: Install conda dependencies
         run: |
           mamba install datacube gdal pycodestyle pylint pytest-cov pytest numexpr lxml pydash python-rapidjson \
-          lxml pydash python-rapidjson ruamel.yaml structlog ciso8601 cattrs boltons dawg
+          lxml pydash python-rapidjson ruamel.yaml structlog ciso8601 cattrs boltons build
           # gdal is required by digitalearthau??
       - name: Install pip deps
         run: |
-          pip install --extra-index-url=https://packages.dea.ga.gov.au/ eodatasets3 digitalearthau
+          pip install --extra-index-url=https://packages.dea.ga.gov.au/ eodatasets3
```

Comment:
Probably don't want to be pulling from packages.dea.ga.gov.au for eodatasets3. I think it's in PyPI and should be grabbed from there.

Reply:
It could even be grabbed from conda-forge alongside `datacube`!